client/app/dbaas/logs/detail/tokens/logs-tokens.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Tokens
```

Comment:
hardcoded values. Will be translated later I guess?

Reply:
This change got into the PR because of merge conflicts. It is not a part of the PR now :)